docs/docs/getting-started/tutorials/local.md

```diff
@@ -2,7 +2,10 @@
 
 ## Prerequisites
 
-JDK 8 or higher
+JDK 8 (newer versions may not be supported)
```

Comment:
It would be better to use the java toolchain support in gradle to define this directly and get rid of these instructions altogether. https://docs.gradle.org/current/userguide/toolchains.html